Simplenote/src/main/java/com/automattic/simplenote/DeepLinkActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.automattic.simplenote;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.automattic.simplenote.utils.AuthUtils;
+
+import net.openid.appauth.RedirectUriReceiverActivity;
+
+public class DeepLinkActivity extends AppCompatActivity {
+    private static final String AUTHENTICATION_SCHEME = "auth";
+    private static final String LOGIN_SCHEME = "login";
```

Comment:
we also have internal links of the form `simplenote://note/NOTEID` - is this important here?

Reply:
Are those only internal for internote, or are supposed to be handled from external apps too? The only deeplink we found in the app was `simplenote://auth?...` used for wpcom login, that's why it's the only one we have here.
The goal of creating `DeepLinkActivity` was making handling different deeplinks easier, by acting as a proxy and dispatching the deeplink to the responsible part.

Reply:
It's my impression they should only be used internally. This question I raise is more from my lack of understanding than from my understanding though. Mainly I wanted to make sure y'all were aware of it and this wasn't limited by oversight.

Reply:
Thanks for bringing it up, we confirmed that it's not related, internal note linking doesn't use deeplink: https://github.com/Automattic/simplenote-android/blob/6d32ae2a2e4f48359598ba4c2fd9cd0d45687d3a/Simplenote/src/main/java/com/automattic/simplenote/utils/SimplenoteLinkify.java#L62-L90